lib/features/children/vpc/pages/give_vpc_page.dart

```diff
@@ -31,13 +32,12 @@ class GiveVPCPage extends StatelessWidget {
                 backgroundColor: Theme.of(context).errorColor,
               ),
             );
-            context.read<VPCCubit>().showProfiles();
+            context.read<VPCCubit>().resetVPC();
```

Comment:
Why do you need to reset? Instead of allowing the bloc to be disposed

Reply:
we need to keep VPCCubit alive to address requirement from KIDS-1141. Long story short user shouldn't go through VPC subflow if it was passed already, while the app is running (so I keep it in the flag just in memory)